geps/gep-2198.md

```diff
@@ -0,0 +1,103 @@
+# JWT Policy
+
+## TLDR
+
+This document is not intended to describe how a proxy implements JWT handling behavior, there is an RFC that addresses that.
+Rather, this document is to form a common understanding or JWT use cases as it relates to configuring and handling JWTs as par of a workflow as it relates to a Gateway and establish a framework from which to build the common parts of the Policy.
+
+## Goals
+
+- Define how JWT is used to form a common understanding and establish a common language
+- Identify unique use cases for JWT with proxies
```

Comment:
Is Oauth or OIDC in scope ?

Reply:
I see OIDC/OAuth2 as its own policy.
It is the "how you obtain a token" part of the process if the proxy is the point of auth.

Reply:
The client can always authenticate before the proxy and include the JWT from the OIDC/OAuth2 IdP with the request rather than have the proxy handle the OIDC workflow.
If the OIDC workflow happens at the proxy then there would be no JWT validation Policy, but there could still be an authorization Policy.  Or an action to transpose token values into headers for backends.